HashMapSort/src/main/java/com/AndrieievYurii/ElementsDistribution.java

```diff
@@ -0,0 +1,36 @@
+package com.AndrieievYurii;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class ElementsDistribution {
```

Comment:
I guess this is not best name for this class. ElementDistribution sounds like some kind of Supplier. But this class doesn`t supply anything, it has particular aim - count amount of unique chars in the string. So give it another more appropriate name